charts/pulsar-operator/templates/pulsar-operator/operator.yaml

```diff
@@ -49,6 +49,7 @@ spec:
         args:
         - --metrics-addr=127.0.0.1:8080
         - --enable-leader-election
+        - --watch-namespace={{ .Values.watchNamespace }}
```

Comment:
If the watch namespace is not set, but the clusterRole is false, the operator should watch its own namespace?

Reply:
> If the watch namespace is not set, but the clusterRole is false, the operator should watch its own namespace?

yeah, my bad. I have did it at the role namespace, forgot to update here.

Reply:
Done.